### PR TITLE
[FrameworkBundle] MicroKernelTrait configureRouting(Configurator)

### PR DESCRIPTION
**Pull request by vudaltsov**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony#32937, https://github.com/symfony/symfony-docs/pull/12688

You can see the differences in commit https://github.com/symfony/recipes/pull/690/commits/aef398a5e8dfa1036009d224d711cad1525ab319

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
The recipe should use as much symlinks as possible referring to files from the previous version. See existing symlinks in the repository.

**Comment by vudaltsov** [comment]
Oops, didn't know that. Done.

File: symfony/framework-bundle/4.2/src/Kernel.php [hunk 1]
```diff
@@ -43,12 +43,12 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $loader->load($confDir.'/{services}_'.$this->environment.self::CONFIG_EXTS, 'glob');
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routes): void
+    protected function configureRouting(RoutingConfigurator $routes): void
```
Comment (fabpot) [inline symfony/framework-bundle/4.2/src/Kernel.php]: The changes in this file look wrong as this is for 4.2.

Comment (vudaltsov) [reply symfony/framework-bundle/4.2/src/Kernel.php]: fixed

**Pull request merged**